app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ViewProspect.php

```diff
@@ -40,101 +40,30 @@
 use Filament\Infolists\Infolist;
 use Filament\Resources\Pages\ViewRecord;
 use AdvisingApp\Prospect\Models\Prospect;
-use Filament\Infolists\Components\Section;
-use Filament\Infolists\Components\IconEntry;
-use Filament\Infolists\Components\TextEntry;
 use AdvisingApp\Prospect\Concerns\ProspectHolisticViewPage;
 use AdvisingApp\Prospect\Filament\Resources\ProspectResource;
 use AdvisingApp\Notification\Filament\Actions\SubscribeHeaderAction;
 use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Actions\ConvertToStudent;
 use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Actions\ProspectTagsAction;
 use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Actions\DisassociateStudent;
+use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Schemas\ProspectProfileInfolist;
+use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Pages\Concerns\HasStudentHeader;
 
 class ViewProspect extends ViewRecord
 {
     use ProspectHolisticViewPage;
+    use HasStudentHeader;
```

Comment:
If this Trait is being shared between Student and Prospect then I think we should rename it to `HasEducatableHeader`

Reply:
As per @danharrin  comment he want `HasProspectHeader` so we don't need to create `HasEducatableHeader`.
Correct? @Orrison

Reply:
@amit-canyon correct. Since you will be making separate Traits for each.